frontend/src/lib/components/apps/editor/AppEditor.svelte

```diff
@@ -297,7 +297,7 @@
 										</div>
 									</Tab>
 								</Popover>
-								<div slot="content" class="h-full overflow-y-auto">
+								<div slot="content" class="relative h-full overflow-y-auto">
```

Comment:
that seems like a big change @fatonramadani

Reply:
Removed it as it wasn't even needed